<details>

<summary>FTR Error from the first PR (from dependabot)</summary>

```
[2021-08-16T06:07:22.947Z] Failed while enforcing RequireUpperBoundDeps. The error(s) are [
[2021-08-16T06:07:22.947Z] Require upper bound dependencies error for org.jenkins-ci.plugins:display-url-api:2.3.4 paths to dependency are:
[2021-08-16T06:07:22.947Z] +-org.jenkins-ci.main:jenkins-test:2.307-rc31419.e84257757bd5
[2021-08-16T06:07:22.947Z]   +-org.jenkins-ci.plugins:mailer:1.34
[2021-08-16T06:07:22.948Z]     +-org.jenkins-ci.plugins:display-url-api:2.3.4
[2021-08-16T06:07:22.948Z] and
[2021-08-16T06:07:22.948Z] +-org.jenkins-ci.main:jenkins-test:2.307-rc31419.e84257757bd5
[2021-08-16T06:07:22.948Z]   +-org.jenkins-ci.plugins:junit:1.52
[2021-08-16T06:07:22.948Z]     +-org.jenkins-ci.plugins:display-url-api:2.3.5
[2021-08-16T06:07:22.948Z] and
[2021-08-16T06:07:22.948Z] +-org.jenkins-ci.main:jenkins-test:2.307-rc31419.e84257757bd5
[2021-08-16T06:07:22.948Z]   +-org.jenkins-ci.plugins:junit:1.52
[2021-08-16T06:07:22.948Z]     +-io.jenkins.plugins:checks-api:1.7.2
[2021-08-16T06:07:22.948Z]       +-org.jenkins-ci.plugins:display-url-api:2.3.5
[2021-08-16T06:07:22.948Z] ]
```


</details>

In this commit, there are some failure (build still running). I compared them with latest commit build on master.

Also failing in latest core commit

- hudson.slaves.SlaveComputerTest

Was **not** failing in latest core commit

- hudson.model.UsageStatisticsTest

```
09:01:13  [ERROR] Tests run: 5, Failures: 1, Errors: 4, Skipped: 0, Time elapsed: 33.093 s <<< FAILURE! - in hudson.model.UsageStatisticsTest
09:01:13  [ERROR] roundtrip(hudson.model.UsageStatisticsTest)  Time elapsed: 24.361 s  <<< FAILURE!
09:01:13  org.junit.ComparisonFailure: expected:<...yleProject":0,"org-j[enkinsci-plugins-workflow-job-WorkflowJob":0,"org-j]vnet-hudson-test-Moc...> but was:<...yleProject":0,"org-j[]vnet-hudson-test-Moc...>
09:01:13  	at org.junit.Assert.assertEquals(Assert.java:117)
09:01:13  	at org.junit.Assert.assertEquals(Assert.java:146)
09:01:13  	at hudson.model.UsageStatisticsTest.compareWithFile(UsageStatisticsTest.java:174)
09:01:13  	at hudson.model.UsageStatisticsTest.roundtrip(UsageStatisticsTest.java:125)
```

Seems some naming change, not investigated deeper

--- a/test/src/test/resources/hudson/model/UsageStatisticsTest/jobs.json
+++ b/test/src/test/resources/hudson/model/UsageStatisticsTest/jobs.json
@@ -1,1 +1,1 @@
-{"com-cloudbees-hudson-plugins-folder-Folder":0,"hudson-matrix-MatrixProject":0,"hudson-model-FreeStyleProject":0,"org-jenkinsci-plugins-workflow-job-WorkflowJob":0,"org-jvnet-hudson-test-MockFolder":0,"org-jvnet-hudson-test-SecuredMockFolder":0}
+{"com-cloudbees-hudson-plugins-folder-Folder":0,"hudson-matrix-MatrixProject":0,"hudson-model-FreeStyleProject":0,"org-jvnet-hudson-test-MockFolder":0,"org-jvnet-hudson-test-SecuredMockFolder":0}

